.github/workflows/buf-pr.yaml

```diff
@@ -0,0 +1,31 @@
+name: Buf PR
+on:
+  pull_request:
+    types: [opened,synchronize,reopened,labeled,unlabeled]
+permissions: read-all
+jobs:
+  buf-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Buf setup
+        uses: bufbuild/buf-setup-action@v1
+        with:
+          version: 1.32.0-beta.1
+      - run: buf --version
+      - name: Buf build
+        run: buf build . --error-format github-actions
+      - name: Buf format
+        if: ${{ !contains(github.event.pull_request.labels.*.name, 'Buf Skip Format') }}
```

Comment:
Thoughts on `Buf Skip Format` vs `Skip Buf Format` vs `Skip buf format` vs `skip buf format`? (same question for other instances)

Reply:
I originally thought Title Case was more consistent with most labels, but lower case would be my preference. Also I think it works better for checks within the merge commit message.

Reply:
going to continue this conversation on https://github.com/bufbuild/buf-action/pull/9